docs/design/mvp-summary-for-client.md

```diff
@@ -0,0 +1,65 @@
+# MVP Scope Summary for Client
+
+## Identity Access Control
+- User accounts
+  - manual creation, modification, and deletion for MVP
+- Staff roles
+  - manual creation, modification, and deletion for MVP
+- IP whitelist (can only access Chronicle from onsite at spa)
+
+## Customer Profiles
+- Core profiles field - creation and modification
+  - name
+  - address
+  - email
+  - phone number
+  - dob
+  - sex and gender
+- Medical record
+  - Weight and height
+  - Allergies
+  - Conditions
+  - Additional Notes
+- Membership level record
+  - Membership purchase and renewal history
+
+## Entries
+- Activity Entry 
+  - activity name
+  - price
+  - date/time
+  - provider
+  - general notes
+  - critical notes
+  - photos
+- Purchase Entry
+  - product name
+  - variant
+  - price
+  - barcode
+  - associated activity entry (if applicable)
+- client creation, deletion, modification
+- Entries will be associated with customers and providers
+- Log of modifications to entries made by staff
+- All searchable by customer, provider and activity/product purchase
+
+## Inventory
+- Treatments 
+  - name
+  - price
+- Physical Products
+  - name
+  - barcode
+  - SKU
+  - variants
+  - prices
+
+- All searchable
+- manual creation, modification, and deletion for MVP
+
+## Data Reporting
+- rigid end of day/week reports 
+  - Staff log in and out 
+  - Activities done
+  - Products purchased
+  - Customer visits
```

Comment:
Some extra details would be nice, maybe in a separate doc.

Reply:
Yeah, this is one area I was thinking we should elaborate for ourselves and we need to write stories for this specifically

Reply:
I double checked with client, those are pretty much the basic one they want for mvp. we should still flush out in our own stories